src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nolinewrap/InputnolinewrapGood.java

```diff
@@ -0,0 +1,17 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace.nolinewrap;
+
+/**
+ *
+ * @author divsidhu-26
```

Comment:
We are expecting files to be nearly a copy/paste of their original. We are not expecting any changes except package name and class name as long as it is not required by test.